docs/user-guide/helm.md

```diff
@@ -28,6 +28,8 @@ spec:
 !!! note "When using multiple ways to provide values"
     Order of precedence is `parameters > valuesObject > values > valueFiles > helm repository values.yaml` (see [Here](./helm.md#helm-value-precedence) for a more detailed example)
 
+For more info about Helm Chart Repositories and how to configure Private Helm repos see [Here](../operator-manual/declarative-setup.md#helm-chart-repositories).
```

Comment:
I'd reword it slightly, otherwise LGTM 👍 

```suggestion
See [here](../operator-manual/declarative-setup.md#helm-chart-repositories) for more info about how to configure private Helm repositories.
```